src/Merchants/Content/Merchant/MerchantEntity.php

```diff
@@ -18,6 +18,16 @@ class MerchantEntity extends Entity
 {
     use EntityIdTrait;
 
+    /**
+     * @var bool
```

Comment:
Laut Definition nicht required koennte also auch null sein

Reply:
Gibs nen Default in der DB auf 0 / false